packages/headless/src/index.ts

```diff
@@ -83,3 +83,5 @@ export {RangeFacetSortCriterion} from './features/facets/range-facets/generic/in
 export {buildSearchParameterSerializer} from './features/search-parameters/search-parameter-serializer';
 export * as HighlightUtils from './utils/highlight';
 export {HighlightKeyword} from './utils/highlight';
+
+export const VERSION = process.env.VERSION!;
```

Comment:
This results in the following:

```js
// dist/headless.js

// 3rd Party Libraries
const VERSION = "0.8.0-alpha.20";

exports.HeadlessEngine = HeadlessEngine;
exports.HighlightUtils = highlight;
exports.TestUtils = index;
exports.VERSION = VERSION;
exports.baseFacetResponseSelector = baseFacetResponseSelector;
exports.buildBreadcrumbManager = buildBreadcrumbManager;
exports.buildCartRecommendationsList = buildCartRecommendationsList;
...
```

Reply:
Would it be possible to place the version on the store?

The issue with a variable is it would be treeshaked away when customers build for production. If instead the version is on the store, the Support team and ourselves will still be able to see the version using redux dev tooling when debugging.

Reply:
I'm going to add the version to the state but I won't remove it from the exports since exports are available everywhere, even in places that don't have a reference to the engine (such as Stencil's global script)